app/src/components/Link.css

```diff
@@ -0,0 +1,3 @@
+.decoration {
+  text-decoration: none;
```

Comment:
do not remove decoration by default 
it is default link behavior. if you dont need it in some places just apply outside style via `className` property 
In most cases you would want to link to be underlined